charts/ratify/templates/configmap.yaml

```diff
@@ -30,10 +30,13 @@ data:
         ]
       },
       "policies": {
-          "version": "1.0.0",
-          "artifactVerificationPolicies": {
-              "application/vnd.cncf.notary.v2.signature": "any"
-          }
+        "version": "1.0.0",
+        "policyPlugin": {
+            "name": "configPolicy",
```

Comment:
nit: as it already in the context of policy, do you need the suffix policy?

Reply:
I'll remove the policy suffix